.github/workflows/intel_configure.sh

```diff
@@ -68,5 +68,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DTRILINOS_LIBRARY_PREFIX="trilinos_" \
 -DCMAKE_UNITY_BUILD=ON
 
-# Buid
+# Build
```

Comment:
Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here: